docs/en/reference/batch-processing.rst

```diff
@@ -82,6 +82,7 @@ with the batching strategy that was already used for bulk inserts:
         $user = $row[0];
         $user->increaseCredit();
         $user->calculateNewBonuses();
+        $em->persist($user);
```

Comment:
not needed here (unless you use explicit change tracking), as the user object is already managed